### ISSUE TITLE
[910B4]Embedding模型返回问题

### ISSUE THREAD

**Issue by JasonFlyBeauty** [xorbitsai/inference#2490]
### System Info / 系統信息

驱动版本：Ascend-hdk-910b-npu-driver_23.0.3_linux-aarch64.run

CANN版本：Ascend-cann-toolkit_8.0.RC3.alpha001_linux-aarch64.run

(xinference) [root@ app]# pip list
Package               Version
--------------------- -----------
accelerate            1.0.1
aiofiles              23.2.1
aioprometheus         23.12.0
annotated-types       0.7.0
anyio                 4.6.2.post1
ascendebug            0.1.0
async-timeout         4.0.3
attrs                 24.2.0
auto-tune             0.1.0
bcrypt                4.2.0
certifi               2024.8.30
cffi                  1.17.1
charset-normalizer    3.4.0
click                 8.1.7
cloudpickle           3.1.0
cryptography          43.0.3
dataflow              0.0.1
decorator             5.1.1
distro                1.9.0
ecdsa                 0.19.0
exceptiongroup        1.2.2
fastapi               0.115.4
ffmpy                 0.4.0
filelock              3.16.1
fsspec                2024.10.0
gradio                5.4.0
gradio_client         1.4.2
h11                   0.14.0
hccl                  0.1.0
hccl-parser           0.1
httpcore              1.0.6
httpx                 0.27.2
huggingface-hub       0.26.1
idna                  3.10
Jinja2                3.1.4
jiter                 0.6.1
joblib                1.4.2
llm-datadist          0.0.1
markdown-it-py        3.0.0
MarkupSafe            2.1.5
mdurl                 0.1.2
ml_dtypes             0.5.0
modelscope            1.19.2
mpmath                1.3.0
msadvisor             1.0.0
networkx              3.4.2
numpy                 1.26.4
nvidia-ml-py          12.560.30
op-compile-tool       0.1.0
op-gen                0.1
op-test-frame         0.1
opc-tool              0.1.0
openai                1.52.2
orjson                3.10.10
packaging             24.1
pandas                2.2.3
passlib               1.7.4
peft                  0.13.2
pillow                11.0.0
pip                   24.2
psutil                6.1.0
pyasn1                0.6.1
pycparser             2.22
pydantic              2.9.2
pydantic_core         2.23.4
pydub                 0.25.1
Pygments              2.18.0
python-dateutil       2.9.0.post0
python-jose           3.3.0
python-multipart      0.0.12
pytz                  2024.2
PyYAML                6.0.2
quantile-python       1.1
regex                 2024.9.11
requests              2.32.3
rich                  13.9.3
rsa                   4.9
ruff                  0.7.1
safehttpx             0.1.1
safetensors           0.4.5
schedule-search       0.0.1
scikit-learn          1.5.2
scipy                 1.14.1
semantic-version      2.10.0
sentence-transformers 3.2.1
setuptools            75.1.0
shellingham           1.5.4
six                   1.16.0
sniffio               1.3.1
sse-starlette         2.1.3
starlette             0.41.2
sympy                 1.13.3
tabulate              0.9.0
tblib                 3.0.0
te                    0.4.0
threadpoolctl         3.5.0
timm                  1.0.11
tokenizers            0.20.1
tomlkit               0.12.0
torch                 2.1.0
torch-npu             2.1.0.post3
torchvision           0.16.0
tornado               6.4.1
tqdm                  4.66.5
transformers          4.46.0
typer                 0.12.5
typing_extensions     4.12.2
tzdata                2024.2
urllib3               2.2.3
uvicorn               0.32.0
uvloop                0.21.0
websockets            12.0
wheel                 0.44.0
xinference            0.16.1
xoscar                0.3.3

### Running Xinference with Docker? / 是否使用 Docker 运行 Xinfernece？

- [ ] docker / docker
- [X] pip install / 通过 pip install 安装
- [ ] installation from source / 从源码安装

### Version info / 版本信息

 printenv
SHELL=/bin/bash
HISTCONTROL=ignoredups
CONDA_EXE=/root/miniconda3/bin/conda
_CE_M=
HISTSIZE=1000
HOSTNAME=135-17
PWD=/home/app
LOGNAME=root
CONDA_PREFIX=/root/miniconda3/envs/xinference
MOTD_SHOWN=pam
HOME=/root
LANG=zh_CN.UTF-8
CONDA_PROMPT_MODIFIER=(xinference) 
SSH_CONNECTION=192.168.0.16 1249 192.168.135.17 22
TOOLCHAIN_HOME=/usr/local/Ascend/ascend-toolkit/latest/toolkit
ASCEND_TOOLKIT_HOME=/usr/local/Ascend/ascend-toolkit/latest
SELINUX_ROLE_REQUESTED=
PYTHONPATH=/usr/local/Ascend/ascend-toolkit/latest/python/site-packages:/usr/local/Ascend/ascend-toolkit/latest/opp/built-in/op_impl/ai_core/tbe:
TERM=xterm
_CE_CONDA=
USER=root
CONDA_SHLVL=2
SELINUX_USE_CURRENT_RANGE=
SHLVL=1
ASCEND_OPP_PATH=/usr/local/Ascend/ascend-toolkit/latest/opp
CONDA_PYTHON_EXE=/root/miniconda3/bin/python
LD_LIBRARY_PATH=/usr/local/Ascend/ascend-toolkit/latest/tools/aml/lib64:/usr/local/Ascend/ascend-toolkit/latest/tools/aml/lib64/plugin:/usr/local/Ascend/ascend-toolkit/latest/lib64:/usr/local/Ascend/ascend-toolkit/latest/lib64/plugin/opskernel:/usr/local/Ascend/ascend-toolkit/latest/lib64/plugin/nnengine:/usr/local/Ascend/ascend-toolkit/latest/opp/built-in/op_impl/ai_core/tbe/op_tiling/lib/linux/aarch64:/usr/local/Ascend/driver/lib64:/usr/local/Ascend/driver/lib64/common:/usr/local/Ascend/driver/lib64/driver:
ASCEND_AICPU_PATH=/usr/local/Ascend/ascend-toolkit/latest
SSH_CLIENT=192.168.0.16 1249 22
CONDA_DEFAULT_ENV=xinference
PATH=/usr/local/Ascend/ascend-toolkit/latest/bin:/usr/local/Ascend/ascend-toolkit/latest/compiler/ccec_compiler/bin:/usr/local/Ascend/ascend-toolkit/latest/tools/ccec_compiler/bin:/root/miniconda3/envs/xinference/bin:/root/miniconda3/condabin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin
SELINUX_LEVEL_REQUESTED=
ASCEND_HOME_PATH=/usr/local/Ascend/ascend-toolkit/latest
MAIL=/var/spool/mail/root
SSH_TTY=/dev/pts/3
CONDA_PREFIX_1=/root/miniconda3
OLDPWD=/root
_=/usr/bin/printenv

### The command used to start Xinference / 用以启动 xinference 的命令

conda create -n xinference python=3.10

conda activate xinference 

pip3 install torch==2.1.0 torchvision==0.16.0 --index-url https://download.pytorch.org/whl/cpu -i https://pypi.tuna.tsinghua.edu.cn/simple

pip3 install pyyaml  -i https://pypi.tuna.tsinghua.edu.cn/simple
pip3 install setuptools -i https://pypi.tuna.tsinghua.edu.cn/simple

pip3 install sentence-transformers -i https://pypi.tuna.tsinghua.edu.cn/simple
pip3 install attrs  -i https://pypi.tuna.tsinghua.edu.cn/simple
pip3 install ml-dtypes tornado -i https://pypi.tuna.tsinghua.edu.cn/simple

pip3 install 'numpy<2.0' -i https://pypi.tuna.tsinghua.edu.cn/simple
pip3 install decorator -i https://pypi.tuna.tsinghua.edu.cn/simple
pip3 install torch-npu==2.1.0.post3 -i https://pypi.tuna.tsinghua.edu.cn/simple

pip3 install xinference -i https://pypi.tuna.tsinghua.edu.cn/simple

xinference-local --host 0.0.0.0 --port 9997

xinference launch --model-name bge-m3 --model-type embedding --gpu-idx 0 --host 0.0.0.0

### Reproduction / 复现过程

curl -X 'POST' \
>   'http://0.0.0.0:9997/v1/embeddings' \
>   -H 'accept: application/json' \
>   -H 'Content-Type: application/json' \
>   -d '{
>     "model": "bge-m3",
>     "input": "What is the capital of China?"
>   }'
{"object":"list","model":"bge-m3-1-0","data":[{"index":0,"object":"embedding","embedding":[0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]}],"usage":{"prompt_tokens":0,"total_tokens":0}}

### Expected behavior / 期待表现

Embedding模型正常回复

**Comment by qinxuye** [comment]
哪个模型？

**Comment by JasonFlyBeauty** [comment]
> 哪个模型？

我们更换了cann Kernels 问题已解决
